core-bundle/src/Resources/contao/classes/FrontendTemplateTrait.php

```diff
@@ -0,0 +1,222 @@
+<?php
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao;
+
+/**
+ * @property integer $id
+ * @property string  $keywords
+ * @property string  $content
+ * @property array   $sections
+ * @property array   $positions
+ * @property array   $matches
+ * @method   string  getTemplate(string $strTemplate)
+ *
+ * @author Leo Feyer <https://github.com/leofeyer>
+ *
+ * @internal
```

Comment:
Not sure if we can make this `@internal`, because they methods have been part of the public API before.

Reply:
I think this is fine because it only disallows using the trait, not the methods:

![Screenshot_20210824_145043](https://user-images.githubusercontent.com/5305677/130619283-fdf1ad50-745e-432f-b151-d81b5fca3799.png)

Reply:
@ausi WDYT?

Reply:
Psalm seems happy, too: https://psalm.dev/r/a343df1b5c

Reply:
Symfony seems to be using it this way too, for example here: https://github.com/symfony/symfony/blob/9a2b25dbfd0af2af50147a3d69b0d3ba93f638b0/src/Symfony/Bridge/PhpUnit/Legacy/ExpectDeprecationTraitForV8_4.php the method `expectDeprecation()` is allowed to be used but not the trait itself via `use`.

So I think it is correct this way.